src/itemObjGen.txt

```diff
@@ -2257,6 +2257,144 @@
     ingredients[]
       name = Rope
       count n = 12
+[]
+  name = Mollusk Walker Body
+  category = walkers
+  crafting[]
+    ingredients[]
+      name = Wood
+      count n = 570
+    ingredients[]
+      name = Rope
+      count n = 12
+    ingredients[]
+      name = Fiber
+      count n = 80
+    ingredients[]
+      name = Fiber Weave
+      count n = 2
+    ingredients[]
+      name = Stone
+      count n = 90
+[]
+  name = Mollusk Walker Legs (1 of 2)
+  category = walkers
+  crafting[]
+    ingredients[]
+      name = Wood
+      count n = 140
+    ingredients[]
+      name = Wood Shaft
+      count n = 18
+[]
+  name = Mollusk Walker Wings (1 of 2)
+  category = walkers
+  crafting[]
+    ingredients[]
+      name = Wood
+      count n = 47
+      name = Fiber Weave
```

Comment:
Missing an ingredient array declaration. Please fix this for all following ingredients as well

Reply:
Doy... I fix one typo and I make others... haha I'll fix it and add the rest of the Mollusk info later today, good catch!